### ISSUE TITLE
Feature request: Disable message about detached clients

### ISSUE THREAD

**Issue by simonpl** [mobile-shell/mosh#559]
If i put one device with a running mosh session to standby and then open a new mosh session (same user) with another device, mosh tells me:

```
    Mosh: You have a detached Mosh session on this server (mosh [pid]).
```

It'd be nice to have the possibility to disable this message.

**Comment by mojoTX** [comment]
If I log in more than once to a server I get this warning. 

I'm running Ubuntu Linux, and compiling from source on the server side, `mosh-1.3.2-55-g6fb4d0d`. The client side is macOS and running mosh 1.3.2 from brew.

As far as I know, a separate `mosh-server` process is required for each connection, and I think that having multiple logins to the same server is not THAT unusual. 

Example, I just logged three times into the same development Linux system:
```
Mosh: You have 2 detached Mosh sessions on this server, with PIDs:
        - mosh [15561]
        - mosh [15706]
```

The other sessions are still alive and well:
```
maj@maj-test-vm:~$ echo $PPID
15561
```
```
maj@maj-test-vm:~/src/GitHub/mosh$ echo $PPID
15706
```

**Comment by eminence** [comment]
HI @mojoTX are you saying that if you have 3 active sessions (meaning connections with running/connect mosh-clients), you are still seeing the "2 detached Mosh sessions" message?  If so, that seems like a bug

**Comment by mojoTX** [comment]
Yes, @eminence, that is the behavior I'm seeing. I frequently launch more than one active sessions to the same Linux server using this command:
```
/usr/local/bin/mosh --ssh="/usr/local/bin/ssh -Ax" user@ipaddress
``` 
On the second and subsequent connection, I start seeing the warning about the detached sessions. 

Everything seems to work fine, but it is mildly irritating.

**Comment by cgull** [comment]
Yup, it's a bug, I see it too.  All sessions are marked as idle, including the one you're logged into.  It's almost certainly related to the recent syslog changes.  I'll look more closely soon.

**Comment by eminence** [comment]
I'd like to move that bug to a new issue, since I think the original feature request here is still valid.

**Comment by mojoTX** [comment]
@eminence and @cgull: Let me know if there's anything I can do to help track down the bug.

**Comment by cgull** [comment]
Ah, right.  I'd lost track of the original feature request while looking at this.

**Comment by cgull** [comment]
Right, so let's continue the detached-sessions conversation on #942.
